metrics/web/test/project_groups/presentation/widgets/delete_project_group_dialog_test.dart

```diff
@@ -325,52 +327,91 @@ void main() {
         await tester.pumpAndSettle();
 
         expect(find.byType(DeleteProjectGroupDialog), findsNothing);
+
+        await tester.pump(DurationConstants.toast);
       },
     );
 
     testWidgets(
       "does not close the dialog if the deleting is finished with an error",
       (tester) async {
-        final notifierMock = ProjectGroupsNotifierMock();
-
-        when(notifierMock.deleteProjectGroupDialogViewModel).thenReturn(
-          deleteProjectGroupDialogViewModel,
-        );
-        when(notifierMock.projectGroupSavingError).thenReturn('error');
+        when(projectGroupsNotifier.projectGroupSavingError).thenReturn('error');
 
         await mockNetworkImagesFor(() {
           return tester.pumpWidget(_DeleteProjectGroupDialogTestbed(
-            projectGroupsNotifier: notifierMock,
+            projectGroupsNotifier: projectGroupsNotifier,
           ));
         });
 
         await tester.tap(deleteButtonFinder);
         await tester.pumpAndSettle();
 
         expect(find.byType(DeleteProjectGroupDialog), findsOneWidget);
+
+        await tester.pump(DurationConstants.toast);
       },
     );
 
     testWidgets(
       "displays the delete project group button text if the deleting is finished with an error",
       (tester) async {
-        final notifierMock = ProjectGroupsNotifierMock();
-
-        when(notifierMock.deleteProjectGroupDialogViewModel).thenReturn(
-          deleteProjectGroupDialogViewModel,
-        );
-        when(notifierMock.projectGroupSavingError).thenReturn('error');
+        when(projectGroupsNotifier.projectGroupSavingError).thenReturn('error');
 
         await mockNetworkImagesFor(() {
           return tester.pumpWidget(_DeleteProjectGroupDialogTestbed(
-            projectGroupsNotifier: notifierMock,
+            projectGroupsNotifier: projectGroupsNotifier,
           ));
         });
 
         await tester.tap(deleteButtonFinder);
         await tester.pumpAndSettle();
 
         expect(deleteButtonFinder, findsOneWidget);
+
+        await tester.pump(DurationConstants.toast);
+      },
+    );
+
+    testWidgets(
+      "displays a negative toast with an error message if an action finished with en error",
+      (tester) async {
+        const message = 'error message';
+        when(projectGroupsNotifier.projectGroupSavingError).thenReturn(message);
+
+        await mockNetworkImagesFor(() {
+          return tester.pumpWidget(_DeleteProjectGroupDialogTestbed(
+            projectGroupsNotifier: projectGroupsNotifier,
+          ));
+        });
+
+        await tester.tap(find.text(ProjectGroupsStrings.delete));
+        await tester.pump();
+
+        expect(find.widgetWithText(NegativeToast, message), findsOneWidget);
+
+        await tester.pump(DurationConstants.toast);
```

Comment:
```suggestion
        await tester.pump(DurationConstants.toast);
```
Can we think of any way of avoiding this? Maybe some changes to the external library?